qutebrowser/config/configdata.py

```diff
@@ -1781,4 +1781,14 @@ def data(readonly=False):
 
     (re.compile(r'^prompt-yes$'), r'prompt-accept yes'),
     (re.compile(r'^prompt-no$'), r'prompt-accept no'),
+
+    (re.compile(r'^tab-close -l$'), r'tab-close --prev'),
+    (re.compile(r'^tab-close --left$'), r'tab-close --prev'),
+    (re.compile(r'^tab-close -r$'), r'tab-close --next'),
+    (re.compile(r'^tab-close --right$'), r'tab-close --next'),
+
+    (re.compile(r'^tab-only -l$'), r'tab-only --prev'),
+    (re.compile(r'^tab-only --left$'), r'tab-only --prev'),
+    (re.compile(r'^tab-only -r$'), r'tab-only --next'),
+    (re.compile(r'^tab-only --right$'), r'tab-only --next'),
```

Comment:
Even though those are pretty obvious, can you please add some test lines to `TestKeyConfigParser.test_migrations` in `tests/unit/config/test_config.py`?